src/OpenLoco/src/Ui.h

```diff
@@ -108,7 +108,7 @@ namespace OpenLoco::Ui
     {
         const float min = 1.0f;
         const float max = 4.0f;
-        const float step = 1.0f;
+        const float step = 0.25f;
```

Comment:
Worth adding a changelog entry for this, and perhaps the hardware backing in general?

Reply:
Will do once the PR is ready